check/test.go

```diff
@@ -194,8 +228,8 @@ func (t testItem) evaluate(s string) *testOutput {
 		result.testResult = !match
 	}
 
-	result.flagFound = match
 	glog.V(3).Info(fmt.Sprintf("flagFound %v", result.flagFound))
```

Comment:
I might be mistaken but I think `flagFound` means "we found a value defined for this setting", however it was defined (audit command or config setting). I think finding it from the environment would be an alternative way of setting this flag, I don't think it needs its own `envFound` flag

Reply:
I agree with this however I think if that is the definition of flagFound it would likely be better to be call it 'found' or something similar (as now its not necessarily finding a flag)